docs/v13/documentation/make-first-prototype/let-user-change-answers.md.njk

```diff
@@ -21,11 +21,12 @@ Open the `juggling-balls.html` file in your `app/views` folder.
 In the `govukRadios` component, add a `value` line, like this:
 
 ```
-{{ govukRadios({
+{% raw %}{{ govukRadios({
   name: "how-many-balls",
   value: data['how-many-balls'],
   fieldset: {
   ...
+{% endraw %}
 ```
```

Comment:
Is it possible to put the `{% raw %}` tags around the backticks? I feel like that would be easier to read.

Reply:
good idea, seems to work